### ISSUE TITLE
[Question] About "ORA-03101: invalid input data for"

### ISSUE THREAD

**Issue by xiaodp** [sijms/go-ora#335]
Hi @sijms ! Really appreciate your effort on this project!  Here's my problem:

Firslty, I create a table that contains various data types use the following SQL
```
CREATE TABLE OT.my_table (
  varchar2_col VARCHAR2(10),
  char_col CHAR(10),
  nchar_col NCHAR(10),
  number_col NUMBER(10, 2),
  integer_col INTEGER,
  smallint_col SMALLINT,
  float_col FLOAT(10),
  date_col DATE,
  timestamp_col TIMESTAMP,
  timestamp_with_time_zone_col TIMESTAMP WITH TIME ZONE,
  timestamp_with_local_time_zone_col TIMESTAMP WITH LOCAL TIME ZONE,
  interval_year_to_month_col INTERVAL YEAR TO MONTH,
  interval_day_to_second_col INTERVAL DAY TO SECOND,
  clob_col CLOB,
  blob_col BLOB,
  bfile_col BFILE,
  long_raw_col LONG RAW,
  raw_col RAW(10)
);
```
And then insert into some values use:
```
INSERT INTO OT.my_table (
varchar2_col, 
char_col, 
nchar_col, 
number_col, 
integer_col, 
smallint_col, 
float_col, 
date_col, 
timestamp_col, 
timestamp_with_time_zone_col, 
timestamp_with_local_time_zone_col, 
interval_year_to_month_col, 
interval_day_to_second_col, 
clob_col,
blob_col,
bfile_col,
long_raw_col,
raw_col)
VALUES (
'hello', 
'world', 
'unicode',
3.14, 
42, 10,
3.14159, 
SYSDATE,
SYSTIMESTAMP,
SYSTIMESTAMP, 
SYSTIMESTAMP, 
INTERVAL '1-2' YEAR TO MONTH, INTERVAL '1 12:30:00' DAY TO SECOND, 
'lorem ipsum',
EMPTY_BLOB(), 
BFILENAME('MY_DIR', 'MY_FILE'), 
EMPTY_BLOB(), 
'DEADBEEF');
```

when I use v2.6.8 to query  via `SELECT * FROM OT.MY_TABL`, there occurs the error `ORA-03101: invalid input data for`

but I switch to v2.5.34,  everything's ok.

**Comment by sijms** [comment]
ok I will re-generate the error and fix

**Comment by sijms** [comment]
the problem related to lob prefetch=pre 
fixed in 2.6.10
